.github/workflows/kind.yml

```
@@ -162,7 +166,7 @@ jobs:
     - name: PolicyAutomation E2E Tests
       run: |
         export GOPATH=$(go env GOPATH)
-        make e2e-test-coverage-policyautomation
+        KUBECONFIG=${PWD}/kubeconfig_hub_e2e make e2e-test-coverage-policyautomation
```

Should this be called with the kubeconfig with controller permissions?

```suggestion
        KUBECONFIG=${PWD}/kubeconfig_hub make e2e-test-coverage-policyautomation
```